apps/condo/domains/miniapp/access/B2BAppPermission.js

```diff
@@ -9,33 +9,44 @@ const { getById } = require('@open-condo/keystone/schema')
 
 const { checkB2BAppAccessRight } = require('@condo/domains/miniapp/utils/accessSchema')
 const { SERVICE } = require('@condo/domains/user/constants/common')
+const { canDirectlyReadSchemaObjects, canDirectlyManageSchemaObjects } = require('@condo/domains/user/utils/directAccess')
 
 /**
  * B2B App permissions may only be read:
```

Comment:
can all employees in organization with b2b context has this permission?

Reply:
I think they can get `read` access, but not in this PR I guess